Makefile

```diff
@@ -653,10 +654,6 @@ ${PROOFTHEORIES}: %_proofs.thy: %.pbf ${BF_GEN_PATH} ${STATICSOURCES} \
 # Preprocessed source files
 ###########################
 
-%.s: %.S ${GENHEADERS} ${STATICHEADERS} | ${DIRECTORIES}
-	@echo " [CPP] $@"
-	$(Q)${CPP} ${CPPFLAGS} -CC -E -o $@ $<
-
```

Comment:
in the incase-sensitive filesystem lowercase .s file and uppercase .S file are same file. this cause .s and .S become a empty file. and I don't see anywhere is using lowercase .s file. So I deleted. tell me if I was wrong.